plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -1039,7 +1039,7 @@ writeDataError(wrkrInstanceData_t *pWrkrData, instanceData *pData, fjson_object
 			DBGPRINTF("omelasticsearch: error creating file content.\n");
 			ABORT_FINALIZE(RS_RET_ERR);
 		}
-		rendered = (char*)fjson_object_to_json_string(ctx.errRoot);
+		rendered = strdup((char*)fjson_object_to_json_string(ctx.errRoot));
```

Comment:
indeed. I spent a few moments asking myself what would be the best solution; but for now I think that the best we can do to fix the issue quickly. We might think about some code re organisation but I believe this can wait.